app/src/main/res/raw/style.css

```diff
@@ -1,6 +1,6 @@
 body, html {
-    background: #221F1F;
-    color: #FFFFFF;
+    background-color: #38383d;
```

Comment:
What are these for?

Reply:
Just matching the background colour to the about page